assessment-service/src/main/java/uk/gov/laa/ccms/caab/assessment/entity/OpaCheckpoint.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.laa.ccms.caab.assessment.entity;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.Lob;
+import jakarta.persistence.MapsId;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+/**
+ * Represents an Oracle Intelligence Advisor Checkpoint.
+ */
+@Entity
+@Table(name = "XXCCMS_OPA_CHECKPOINT")
+@Data
+public class OpaCheckpoint {
+
+  @Id
+  @Column(name = "RESUME_ID")
+  private long resumeId;
```

Comment:
Should be Long. And could we just call it 'id'?

Reply:
Amended